src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,11 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery ball1 = new Lottery();
+        System.out.println(ball1.getRandomBall().toString());
+        Lottery ball2 = new Lottery();
+        System.out.println(ball2.getRandomBall().toString());
+        Lottery ball3 = new Lottery();
+        System.out.println(ball3.getRandomBall().toString());
```

Comment:
```suggestion
        for (int i = 0; i < 3; i++) {
            Lottery lottery = new Lottery();
            System.out.println(ball1.getRandomBall())
        }
```
Using For-loop prevents you from copying the same code (in this case 3 times)
The constructor of class Lottery gives a Lottery object not a ball, that's why the name of the variable should be showing it. Use the right names to make your code more understandable 
Also when invoking System.out.println(something), toString() method is invoked automatically